File: src/main/java/top/zbeboy/isy/web/vo/data/academic/AcademicTitleVo.java
package top.zbeboy.isy.web.vo.data.academic;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by zbeboy on 2017/4/21.
 */
@Data
public class AcademicTitleVo {
    private Integer academicTitleId;
    @NotNull
    @Size(max = 30)
    private String  academicTitleName;
}

File: src/main/java/top/zbeboy/isy/web/vo/data/academic/AcademicTitleVo.kt
package top.zbeboy.isy.web.vo.data.academic

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2017-12-12 .
 **/
open class AcademicTitleVo {
    var academicTitleId: Int? = null
    @NotNull
    @Size(max = 30)
    var academicTitleName: String? = null
}